### PR TITLE
Web UI Changes

### PR DESCRIPTION
**Pull request by pxwxnvermx**
[Ticket](https://dimagi.atlassian.net/browse/CCCT-233)

- Remove auto-generated usernames from Learn, Deliver, User, and Payment Status tables.
- Replaced date times with a popup that shows detailed time information on hovering.
- Clicking on the username popup icon copies the username to the system clipboard.
- Added duration in minutes and seconds on learn progress

image

image

### CODE REVIEW

File: commcare_connect/static/js/vendors.js [hunk 1]
```diff
@@ -1,7 +1,23 @@
 import '@popperjs/core';
-import 'bootstrap';
+import * as bootstrap from 'bootstrap';
 import 'htmx.org';
 import Alpine from 'alpinejs';
 
 window.Alpine = Alpine;
 Alpine.start();
+
+function refreshTooltips() {
```
Comment (calellowitz) [inline commcare_connect/static/js/vendors.js:9]: what does this function do?

**Review by calellowitz — CHANGES_REQUESTED** [review]
One question about the javascript, but the only big thing is that we wanted to get rid of the username column, rather than replace it with a tooltip.

File: commcare_connect/static/js/vendors.js (hunk 1 above)
Comment (pxwxnvermx) [reply commcare_connect/static/js/vendors.js:9]: This function is used for loading the tooltips on page load or table load in this case.

Comment (calellowitz) [reply commcare_connect/static/js/vendors.js:9]: why is it in `vendors/js`? this file seems to be around importing third party libraries

**Review by calellowitz — APPROVED** [review]
I think some of those functions are potentially in the wrong place, but nothing blocking

File: commcare_connect/static/js/vendors.js (hunk 1 above)
Comment (pxwxnvermx) [reply commcare_connect/static/js/vendors.js]: Fixed. https://github.com/dimagi/commcare-connect/pull/252/commits/b5f33d9d99fea130444bce51fe5e71d09340100b

**Pull request merged**